server/src/main/java/org/jboss/as/server/deployment/module/descriptor/DeploymentStructureDescriptorParser.java

```diff
@@ -135,8 +136,12 @@ public void deploy(final DeploymentPhaseContext phaseContext) throws DeploymentU
                 for(AdditionalModuleSpecification i : deploymentUnit.getParent().getAttachmentList(Attachments.ADDITIONAL_MODULES)) {
                     additionalModules.add(i.getModuleIdentifier());
                 }
-                handleDeployment(phaseContext, deploymentUnit, subModuleSpec, deploymentRoot.getAttachment(SUB_DEPLOYMENT_STRUCTURE), additionalModules);
-            }
+                try {
+handleDeployment(phaseContext, deploymentUnit, subModuleSpec, deploymentRoot.getAttachment(SUB_DEPLOYMENT_STRUCTURE), additionalModules);
+} catch (StartException e) {
```

Comment:
This would also need to be correctly formatted.

Reply:
Apologies for late reply,are there just formatting changes which are required?Is there anything else code specific which also needs revisiting?Or should I proceed with fixing the formatting for now?